jwst/pipeline/engdblog.cfg

```diff
@@ -0,0 +1,2 @@
+name = "engdblog"
+class = "jwst.engdblog.EngDBLogStep"
```

Comment:
The default .cfg file in the repo should also include the list of optional arguments (the items in the "spec" list in the step module) set to their default values. That way a user can easily modify the argument settings if they want. For example: https://github.com/STScI-JWST/jwst/blob/master/jwst/pipeline/ramp_fit.cfg